app/Http/Controllers/Api/V1/DataAccessApplicationController.php

```diff
@@ -79,7 +79,12 @@ public function store(CreateDataAccessApplication $request): JsonResponse
                 'project_title' => $input['project_title'],
             ]);
 
-            $submissionStatus = isset($input['submission_status']) ? $input['submission_status'] : 'DRAFT';
```

Comment:
`$submissionStatus = $input['submission_status']) ?? 'DRAFT';`

Reply:
That's a deleted line but I have updated in other places.